### PR TITLE
Add comment submit functionality

### PR DESCRIPTION
**Pull request by andy7809**
This PR adds comment submit/display functionality to my webpage.

### CODE REVIEW

File: portfolio/src/main/webapp/scripts/main.dart [hunk 1]
```diff
@@ -2,13 +2,20 @@
 // dynamic content management/creation.
 
 import 'dart:html';
+import 'dart:convert';
 
-final String collapsibleQueryString = '.collapsible';
+final String COLLAPSIBLE_QUERY_STR = ".collapsible";
```
[the remark this answers is not in the document]
Comment (andy7809) [reply portfolio/src/main/webapp/scripts/main.dart:7]: You're right, I'll change these

[the remark this answers is not in the document]
Comment (andy7809) [reply portfolio/src/main/webapp/scripts/main.dart]: fixed in f0469ce

File: portfolio/src/main/webapp/scripts/main.dart [hunk 2]
```diff
@@ -18,11 +25,36 @@ void addCollapsibleClickListener(Element collapsible) {
...
     contentElement.style.display = "block";
   }
-}
\ No newline at end of file
+}
+
+// Handles a click on the submit button
+void submitComment(Event event) {
+  var commentTextArea = querySelector("#comment") as TextAreaElement; 
+  var commentVal = commentTextArea.value;
```
[the remark this answers is not in the document]
Comment (andy7809) [reply portfolio/src/main/webapp/scripts/main.dart:40]: good idea, will do

[the remark this answers is not in the document]
Comment (andy7809) [reply portfolio/src/main/webapp/scripts/main.dart]: fixed in eb39638

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -60,6 +60,7 @@ private static String objToJson(Object o){
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     try{
       String requestBody = getRequestBody(request);
+      System.out.println(requestBody);
```
Comment (saileshsimhadri) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:63]: Do you still need this?

Comment (andy7809) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:63]: Nope, good catch, thank you

Comment (andy7809) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: fixed in 6ac2bd2

**Pull request merged**